server/player/player.go

```diff
@@ -2747,9 +2747,9 @@ func (p *Player) addNewItem(ctx *item.UseContext) {
 	}
 }
 
-// canReach checks if a player can reach a position with its current range. The range depends on if the player
+// CanReach checks if a player can reach a position with its current range. The range depends on if the player
 // is either survival or creative mode.
-func (p *Player) canReach(pos mgl64.Vec3) bool {
+func (p *Player) CanReach(pos mgl64.Vec3) bool {
```

Comment:
I'm hesitant on making this an exported function. I think it'd be better to keep this unexported and do this check in the EditSign method so we can keep this as it was.